smart_control/reinforcement_learning/scripts/populate_starter_buffer.py

```diff
@@ -78,9 +77,6 @@ def populate_replay_buffer(
 
   # Create policy for collection
   train_step = tf.Variable(0, trainable=False, dtype=tf.int64)
-  observation_spec, action_spec, time_step_spec = spec_utils.get_tensor_specs(
```

Comment:
the `observation_spec` variable is unused, but we can't get rid of this entire function call because it also returns the `action_spec` and `time_step_spec`, which are indeed used. so let's use a `_` to replace just the unused `observation_spec` variable (without breaking the code 😅 ) 

good catch!